docs/book/src/reference/creating-events

```diff
@@ -0,0 +1,109 @@
+# Creating Events
+
+It is often useful to publish *Event* objects from the controller Reconcile function. 
+
+Events allow users to see what is going on with a particular object, and allow automated processes to see and respond to them.
+
+{% panel style="success", title="Getting Events" %}
+Recent Events for an object can be viewed by running `kubectl describe`
+{% endpanel %}
```

Comment:
We should use the same syntax and CSS that is applied across the project in all docs. 
Can you please check it out? Why are you using `{% panel style="success", title="Getting Events" %}`?

Reply:
As mentioned I have just copy pasted the v1 docs and it comes with that. I will surely remove it.

Reply:
I think you want me to remove method end method, panel syntax as well.

Reply:
Check the current docs, see that you can use a search in your IDE or here in the repo to look for the fragment text and find the places so that you know how to do the same.

Reply:
I have observed the current docs. not more css is used. hardly one or two syntax for notes and code snippets. Therefore I have removed all the syntax.